packages/react/src/components/Checkbox/Checkbox.spec.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { axe } from 'jest-axe';
+import { Checkbox } from './Checkbox';
+import userEvent from '@testing-library/user-event';
```

Comment:
does this support hover and focus users events? may be nice to have some ode coverage around those two events.